src/intrinsics.cr

```diff
@@ -1,31 +1,57 @@
 # Intrinsics as exported by LLVM.
 # Use `Intrinsics` to have a unified API across LLVM versions.
 lib LibIntrinsics
+  {% if flag?(:interpreted) %} @[Primitive(:interpreter_intrinsics_debugtrap)] {% end %}
```

Comment:
Since this looks like a pretty common idiom, perhaps it's worth adding a specific annotation, like an `InterpreterPrimitive` to avoid `{% if flag?(:interpreted) %}` boilerplate?

Reply:
It's a good point, but semantic analysis runs on interpreted code which means regular Crystal will have to know about this new annotation. The way it's done now, all regular primitives are primitives in the interpreter, but some regular methods need to be primitives in the interpreter too. I think it's fine.